src/components/AudioPlayer/Player/Player.component.js

```diff
@@ -2,15 +2,18 @@ import React from 'react';
 import propTypes from 'prop-types';
 
 import {
-  Repeat, Rewind, Play, Forward, Shuffle,
+  Repeat,
+  Rewind,
+  PlayContainer,
+  Forward, Shuffle,
 } from '../PlayerButtons';
 
-const PlayerComponent = ({ isDisabled }) => (
+const PlayerComponent = ({ isDisabled, audioRef }) => (
   <div className={`buttons ${isDisabled ? 'disable-buttons' : ''}`}>
     <div className="buttons-container">
       <Repeat />
       <Rewind />
-      <Play />
+      <PlayContainer audioRef={audioRef} />
```

Comment:
IMO this would be better to have the prefix `Button` ie: `PlayButtonContainer` over these components, I know it's under the `PlayerButtons` package but... `PlayContainer` it's a very generic name and finding this could be somewhat hard for another engineer?

Thoughts? @tonymtz @ImAriel2D

Reply:
That's a good idea, i'll add it to the jira board 😄

Reply:
I think the structure we are creating here is over-complicated for this audio player :)